lk2nd/device/dts/msm8953/msm8953-xiaomi-common.dts

```diff
@@ -37,6 +37,7 @@
 		/* Xiaomi Redmi 6 Pro (sakura) and Xiaomi Mi A2 Lite (daisy) are software compatible */
 		model = "Xiaomi Redmi 6 Pro";
 		compatible = "xiaomi,sakura";
+		dtb_compatible = "xiaomi,daisy";
```

Comment:
Can we just put daisy into the array of compatible values? This would indeed make the matching logic a bit more complicated since one'd need to iterate over all compatibles, but the compatible in the line above was kinda meant for that already. I'd strongly prefer to not have two props that do the same thing.